1.16.5/src/main/java/io/github/axolotlclient/modules/hypixel/bedwars/upgrades/BedwarsTeamUpgrades.java

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright © 2021-2023 moehreag <moehreag@gmail.com> & Contributors
+ *
+ * This file is part of AxolotlClient.
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * For more information, see the LICENSE file.
+ */
+
+package io.github.axolotlclient.modules.hypixel.bedwars.upgrades;
+
+
+import java.util.regex.Pattern;
+
+import io.github.axolotlclient.AxolotlClientConfig.Color;
+
+/**
+ * @author DarkKronicle
+ */
+
+public class BedwarsTeamUpgrades {
```

Comment:
Would it be possible to check for a texture within your resource pack and then default to vanilla ones if none are found? This would allow customizing via a resource pack.

Reply:
already possible by default, this implementation is enough for that